plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -172,6 +174,38 @@ public void testV2TableWithEqualityDelete()
         assertQuery("SELECT nationkey FROM " + tableName, "SELECT nationkey FROM nation WHERE regionkey != 1");
     }
 
+    @Test
+    public void testUpgradeTableToV2FromTrino()
+    {
+        String tableName = "test_upgrade_table_to_v2_from_trino_" + randomTableSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " WITH (format_version = 1) AS SELECT * FROM tpch.tiny.nation", 25);
+        assertEquals(loadTable(tableName).operations().current().formatVersion(), 1);
+        assertUpdate("ALTER TABLE " + tableName + " SET PROPERTIES format_version = 2");
+        assertEquals(loadTable(tableName).operations().current().formatVersion(), 2);
```

Comment:
Can we please verify via 

```
assertQuery("SELECT * FROM " + tableName, "SELECT * FROM nation");
```

that the change of the `format_version` has no negative outcome for the end user?

Reply:
Added

And no, I don't think it does

Reply:
I was just thinking that it would be good to have it in case of eventual regressions.